src/i18n.js

```diff
@@ -76,7 +78,15 @@ i18n
       },
       supportedLngs: LANGUAGE_OPTIONS.map((option) => option.value),
       lowerCaseLng: true,
-      fallbackLng: 'en',
+      fallbackLng: {
+        'zh-cn': ['zh-hans', 'en'],
+        'zh-my': ['zh-hans', 'en'],
+        'zh-sg': ['zh-hans', 'en'],
+        'zh-hk': ['zh-hant', 'en'],
+        'zh-mo': ['zh-hant', 'en'],
+        'zh-tw': ['zh-hant', 'en'],
+        default: ['en'],
```

Comment:
zh-hans and zh-hant are provided, not sure why you're providing the second fallback here?

Reply:
Because if a custom fallback is provided, the default is not applied. So without this, you get `fr-fr` → `[fr, en]` but `zh-tw` → `[zh-hant]`.